FlowCrypt/src/main/java/com/flowcrypt/email/database/FlowCryptRoomDatabase.kt

```diff
@@ -512,6 +517,34 @@ abstract class FlowCryptRoomDatabase : RoomDatabase() {
       }
     }
 
+    /**
+     * Here we do preparation for https://github.com/FlowCrypt/flowcrypt-android/issues/1188
+     */
+    @VisibleForTesting
+    val MIGRATION_26_27 = object : FlowCryptMigration(26, 27) {
+      override fun doMigration(database: SupportSQLiteDatabase) {
+        database.execSQL("CREATE TEMP TABLE IF NOT EXISTS contacts_temp AS SELECT * FROM contacts;")
+        database.execSQL("DROP TABLE IF EXISTS contacts;")
+
+        //create `recipients` table
+        database.execSQL("CREATE TABLE IF NOT EXISTS `recipients` (`_id` INTEGER PRIMARY KEY AUTOINCREMENT, `email` TEXT NOT NULL, `name` TEXT DEFAULT NULL, `last_use` INTEGER NOT NULL DEFAULT 0)")
```

Comment:
What does last_use field contain?

Reply:
I think this one has the last time that the we sent email to the recipient. Which we then use for ordering recipients when rendering them.

So that when I enter `ivan` it will generally find your email and put it on top of recipient list for me to choose (because I emailed you most recently), instead of other `ivan`s that I may have written a long time ago.

Reply:
> I think this one has the last time that the we sent email to the recipient. Which we then use for ordering recipients when rendering them.

Exactly